docs/ko-kr/_sidebar.md

```diff
@@ -1,3 +1,46 @@
 - 소개
 
-  - [시작하기](ko-kr/gettingstarted.md)
+  - [Getting Started](/ko-kr/gettingstarted.md)
```

Comment:
can we please also translate the text in the sidebar ("Getting Started", "Why Bloc?", etc...)?